common_design_subtheme/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "common-design-subtheme",
-  "version": "9.0.1",
-  "description": "OCHA Common Design sub theme for Drupal 9+",
+  "name": "common-design",
+  "version": "9.2.0",
+  "description": "OCHA Common Design base theme for Drupal 9+",
```

Comment:
Whoops! I've been forgetting to bump the version recently. Keep the new version, but the sub-theme should probably be different name/desc than the base-theme